docs/adr/0000-database-upgrade-testing.md

```diff
@@ -0,0 +1,60 @@
+# Database Upgrade Testing
+
+* Status: accepted
+* Deciders: [Sander, Arnaud, Wouter]
+* Date: 12/01/2023
```

Comment:
The template contains a date format, I think it could get confusing if we start using different ones across ADRs. I have no objection against changing the format in the template either if you prefer.
```suggestion
* Date: 2023-01-12
```